### ISSUE TITLE
Replace libPusher

### ISSUE THREAD

**Issue by dannygreg** [play/play-cocoa#6]
It is responsible for a dumb leak we had to put in to avoid a crash. We could write a replacement atop AFNetworking fairly trivially.

**Comment by markrickert** [comment]
@samsoffes recently wrote a Pusher library for iOS... maybe could repurpose it here?

https://github.com/samsoffes/bully

**Comment by dannygreg** [comment]
Yep that is absolutely the plan. I spoke to Sam recently about it. I just need to get around to hacking it in. Feel free to submit a pull request :)

**Comment by markrickert** [comment]
Yeah, well I gotta get my @play installation all up and running for my company.

Any advice on setting it up on a mac mini in the office and then punching a hole in the network to give it a canonical domain name so that outsiders can access it?

**Comment by markrickert** [comment]
I worked on a  project a few months ago and shipped an iOS app with the "official" libPusher library in it and it has all sorts of problems :(
